pyproject.toml

```diff
@@ -110,132 +104,123 @@ packages = ["metricflow", "metricflow-semantics/metricflow_semantics"]
 
 [tool.hatch.envs.dev-env]
 description = "Environment for development. Includes a DuckDB-backed client."
-
-pre-install-commands = [
-  "pip install -e ./metricflow-semantics",
-  "pip install -e ./dbt-metricflow[duckdb]",
-]
+run = "run-coverage --no-cov"
+dependencies = ["metricflow-semantics @ {root:uri}/metricflow-semantics"]
 
 features = [
   "dev-packages",
   "sql-client-packages",
+  "dbt-duckdb",
 ]
 
 [tool.hatch.envs.dev-env.env-vars]
 MF_TEST_ADAPTER_TYPE="duckdb"
 MF_SQL_ENGINE_URL="duckdb://"
+# This allows us to use the classes in the `dbt-metricflow` package for tests without installing the package.
+# `dbt-metricflow` can't be installed as it has `metricflow` as a dependency.
+PYTHONPATH="./dbt-metricflow"
```

Comment:
Oh, is this so the lint/test runner will use the local package first instead of a build in the environment install PYTHONPATH?

I wonder if we can use this trick at all for metricflow-semantics, if it turns out the current local dev installs are not editable.

Reply:
Done upstack!